src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1513,6 +1513,23 @@ private String getPortFromMessageTokens(String[] messageTokens) throws PluginExe
         return null;
     }
 
+    private String findLocalPort(String internalContainerPort) {
+        String dockerPortCmd = "docker port " + containerName + " " + internalContainerPort;
+        String cmdResult = execDockerCmd(dockerPortCmd, 10, false);
+        if (cmdResult == null) {
+            error("Unable to retrieve locally mapped port.");
+            return null;
+        }
+        String[] cmdResultSplit = cmdResult.split(":");
+        if (cmdResultSplit[0].equals("Error")) {
```

Comment:
It would be more reliable to check if the return code is 1 instead of looking for the word "Error"